dawn/src/dawn/SIR/SIR.h

```diff
@@ -284,11 +284,11 @@ struct Value : NonCopyable {
 
   template <class T>
   explicit Value(T&& value)
-    : value_{value}, is_constexpr_{false}, type_{TypeInfo<T>::Type} {}
+      : value_{std::forward<T>(value)}, is_constexpr_{false}, type_{TypeInfo<T>::Type} {}
 
   template <class T>
   explicit Value(T value, bool is_constexpr)
-    : value_{value}, is_constexpr_{is_constexpr}, type_{TypeInfo<T>::Type} {}
+      : value_{value}, is_constexpr_{is_constexpr}, type_{TypeInfo<T>::Type} {}
```

Comment:
I think there's no reason not to use the forwarding constructor here too (also for consistency), at least if you don't plan to replace this anyway in the next PR.